packages/app/src/app/hooks/useWorkspaceFeatureFlags.ts

```diff
@@ -18,6 +18,6 @@ export const useWorkspaceFeatureFlags = (): FeatureFlags => {
   }
 
   return {
-    ubbBeta: activeTeamInfo.featureFlags.ubbBeta,
+    ubbBeta: !activeTeamInfo.featureFlags.ubbBeta,
```

Comment:
Remove this before merge

Reply:
Remember this

![image](https://github.com/codesandbox/codesandbox-client/assets/4838076/9ecb95fa-6e6a-4ff3-815c-65dd1c3f64fb)